src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -52,47 +53,17 @@ && arePrefixesPresent(argMultimap, PREFIX_DESCRIPTION)) {
                 && !arePrefixesPresent(argMultimap, PREFIX_DESCRIPTION)) {
             throw new ParseException("Description cannot be empty!");
         } else {
-            String[] category = null;
-            String[] description = null;
-            if (argMultimap.getValue(PREFIX_CATEGORY).isPresent()) {
-                String categoryString = argMultimap.getValue(PREFIX_CATEGORY).get().trim();
-                if (!categoryString.isEmpty()) {
-                    category = categoryString.split(",");
-                } else {
-                    throw new ParseException(AddCategoryCommand.ENTRY_NOT_ADDED);
-                }
-            }
-            if (argMultimap.getValue(PREFIX_DESCRIPTION).isPresent()) {
-                String descriptionString = argMultimap.getValue(PREFIX_DESCRIPTION).get().trim();
-                if (!descriptionString.isEmpty()) {
-                    description = descriptionString.split(",");
-                } else {
-                    throw new ParseException(AddCategoryCommand.ENTRY_NOT_ADDED);
-                }
+            List<String> categories = argMultimap.getAllValues(PREFIX_CATEGORY);
```

Comment:
Great use of `List<String> categories = argMultimap.getAllValues(PREFIX_CATEGORY);` I use it too!